src/points/PointsDiscoverCard.tsx

```diff
@@ -36,63 +36,68 @@ export default function PointsDiscoverCard() {
   }
 
   return (
-    <View style={styles.container}>
-      <Touchable
-        style={styles.touchable}
-        borderRadius={Spacing.Smallest8}
-        onPress={handlePress}
-        testID="PointsDiscoverCard"
-      >
-        <>
-          <View style={styles.header}>
-            <Text style={styles.title}>{t('points.discoverCard.title')}</Text>
-            <View style={styles.pill}>
-              <Text style={styles.balance}>{pointsBalance}</Text>
-              <LogoHeart size={Spacing.Regular16} />
-            </View>
-          </View>
+    <Touchable
+      shouldRenderRippleAbove
+      style={styles.touchable}
+      borderRadius={Spacing.Smallest8}
+      onPress={handlePress}
+      testID="PointsDiscoverCard"
+    >
+      <>
+        <Image style={styles.image} source={pointsCardBackground} />
+        <View style={styles.content}>
+          <Text style={styles.title}>{t('points.discoverCard.title')}</Text>
           <Text style={styles.description}>{t('points.discoverCard.description')}</Text>
-        </>
-      </Touchable>
-    </View>
+          <View style={styles.pill}>
```

Comment:
When testing this out, for some reason I expected the pill to be independently tappable apart from the card as a whole (like you tap it and it highlights just the pill). Doesn't matter since the whole card is tappable, but I guess my instinct with a pill like that is that it itself is a tappable button :sweat_smile:

Reply:
yeah, it definitely looks like a button. let me ask Kayla about that.

Reply:
so she thinks it is not necessary for this pill behave as a button:  https://valora-app.slack.com/archives/C0684TXDR3K/p1719941062683099?thread_ts=1719940711.678299&cid=C0684TXDR3K

my choice is to not add its own ripple, to keep the code simple.

Reply:
okay! sounds good to me :)